dask/multiprocessing.py

```diff
@@ -138,19 +138,9 @@ def pack_exception(e, dumps):
 
 def get_context():
     """ Return the current multiprocessing context."""
-    if sys.platform == "win32":
-        # Just do the default, since we can't change it:
-        if config.get("multiprocessing.context", None) is not None:
-            warn(_CONTEXT_UNSUPPORTED, UserWarning)
-        return multiprocessing
-    context_name = config.get("multiprocessing.context", None)
-    # The default context on OSX was switched from "fork" to "spawn" in
-    # Python 3.8. We keep "fork" as the default here for backwards
-    # compatibility and to avoid raising a RuntimeError when using the
-    # multiprocessing scheduler in a Python script that is not in a
-    # if __name__ == "__main__" block
-    if context_name is None and sys.platform == "darwin":
-        context_name = "fork"
+    # fork context does fork()-without-exec(), which can lead to deadlocks,
+    # so default to "spawn".
+    context_name = config.get("multiprocessing.context", "spawn")
```

Comment:
Ah ok. Seeing the test failure maybe we should just readd the previous Windows check after this line (tweaking it to check if it is not `"spawn"` instead).